app/main.py

```diff
@@ -16,4 +16,35 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+
+variables = {
+    "lucky_number": lucky_number,
+    "pi": pi,
+    "one_is_a_prime_number": one_is_a_prime_number,
+    "name": name,
+    "profile_info": profile_info,
+    "my_favourite_films": my_favourite_films,
+    "marks": marks,
+    "collection_of_coins": collection_of_coins,
+}
+
+mutable_types = (list, dict, set)
```

Comment:
The `mutable_types` tuple should include `set` instead of `frozenset`, as `set` is mutable and `frozenset` is immutable. Ensure that the correct types are used for classification.